packages/collector/test/tracing/cloud/gcp/pubsub/test.js

```diff
@@ -16,24 +17,43 @@ const config = require('../../../../../../core/test/config');
 const { delay, expectExactlyOneMatching, retry, stringifyItems } = require('../../../../../../core/test/test_util');
 const ProcessControls = require('../../../../test_util/ProcessControls');
 const globalAgent = require('../../../../globalAgent');
-
-const projectId = process.env.GCP_PROJECT;
 // We use different topics/subscriptions per Node.js major version so tests on CI run independently of each other.
 const defaultTopicName = `nodejs-test-topic-${semver.parse(process.version).major}`;
 const defaultSubscriptionName = `nodejs-test-subscription-${semver.parse(process.version).major}`;
 
 // This suite is skipped if no GCP project ID has been provided via GPC_PROJECT. It also requires to either have GCP
 // default credentials to be configured, for example via GOOGLE_APPLICATION_CREDENTIALS, or (for CI) to get
 // the credentials as a string from GOOGLE_APPLICATION_CREDENTIALS_CONTENT.
+
+/**
+ * This suite is skipped if no GCP project ID has been provided via GPC_PROJECT. It also requires to either have GCP
+ * default credentials to be configured, for example via GOOGLE_APPLICATION_CREDENTIALS, or (for CI) to get
+ *  the credentials as a string from GOOGLE_APPLICATION_CREDENTIALS_CONTENT.
+ *
+ * https://console.cloud.google.com/home/dashboard?project=k8s-brewery&pli=1
+ *
+ * If you do not have the "google-cloud-platform-k8s-brewery-team-nodejs-service-account-key.json",
+ * please take a look in 1password.
+ */
+if (!process.env.CI && !process.env.GCP_PROJECT) {
+  process.env.GCP_PROJECT = 'k8s-brewery';
+}
+
+if (!process.env.CI && !process.env.GOOGLE_APPLICATION_CREDENTIALS) {
+  process.env.GOOGLE_APPLICATION_CREDENTIALS = path.join(
+    os.homedir(),
+    '.google-cloud-platform-k8s-brewery-team-nodejs-service-account-key.json'
+  );
+}
+
 let mochaSuiteFn;
+const projectId = process.env.GCP_PROJECT;
 
```

Comment:
If we go back to skipping the suite when GCP access has not been configured and running the tests locally, we could add something like this to make sure we do not accidentally skip the suite on CI. (For example if we set up a new CI environment and forget to set GCP_PROJECT and/or GOOGLE_APPLICATION_CREDENTIALS_CONTENT).

```
if (!projectId && process.env.CI) {
  describe(() => {
    it('configuration for Google Cloud Platform is missing', () => {
      fail('Please set GCP_PROJECT and GOOGLE_APPLICATION_CREDENTIALS_CONTENT to enable GCP tests');
    });
  });

  return;

}
```